examples/pytorch/ogc/ogc.py

```diff
@@ -0,0 +1,103 @@
+import time
+import argparse
+import scipy.sparse as sp
+
+import torch
+import torch.nn.functional as F
+
+from dgl import AddSelfLoop
+from dgl.data import CiteseerGraphDataset, CoraGraphDataset, PubmedGraphDataset
+from utils import sparse_mx_to_torch_sparse_tensor, symmetric_normalize_adjacency, LinearNeuralNetwork
+
+
+# Training settings
+decline = 0.9            # the dcline rate
```

Comment:
nit, typo.
start with uppercase and end with dot.

Reply:
same for others.

Reply:
Thanks. We've adjusted all our annotations in accordance with the style guide.